### ISSUE TITLE
runtime error related to github name

### ISSUE THREAD

**Issue by JonasPf** [amitsaha/gitbackup#30]
When I run gitbackup against github I get the following runtime error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x128b525]

goroutine 1 [running]:
main.getUsername(0x13208c0, 0xc0000ee280, 0x7ffeefbff85f, 0x6, 0x13208c0, 0xc0000ee280)
	/Users/jonas/projects/temp/gitbackup/helpers.go:23 +0xd5
main.main()
	/Users/jonas/projects/temp/gitbackup/main.go:53 +0x5a4
exit status 2
```

Looking at the code, it appears that `*user.Name` causes a nil pointer dereference. I changed line 23 in `helpers.go` to use `*user.Login` instead and it seems to fix the issue:

```
	if service == "github" {
		ctx := context.Background()
		user, _, err := client.(*github.Client).Users.Get(ctx, "")
		if err != nil {
			log.Fatal("Error retrieving username", err.Error())
		}
		return *user.Login
	}
```

**Comment by amitsaha** [comment]
Hi @JonasPf thanks for reporting the issue. Can you please paste me the command you ran?

**Comment by JonasPf** [comment]
In the freshly checkout out repo I ran:

```
go run . -service github -backupdir ~/Backups/gitbackup/
```

**Comment by amitsaha** [comment]
I believe you fixed this in PR #36 ?

**Comment by JonasPf** [comment]
indeed, sorry for late response
